root@masterhost:~/ansible_modules_redirection_test4/ansible-junos-stdlib/ansible_collections# ANSIBLE_DEBUG=1 ansible-playbook -i inventory device_facts.yml 538841 1742797546.50788: starting run
538841 1742797546.51512: Added group all to inventory
538841 1742797546.51520: Added group ungrouped to inventory
538841 1742797546.51529: Group all now contains ungrouped
538841 1742797546.51534: Examining possible inventory source: /root/ansible_modules_redirection_test4/ansible-junos-stdlib/ansible_collections/inventory
538841 1742797546.63865: trying /usr/local/lib/python3.12/site-packages/ansible/plugins/cache
538841 1742797546.63942: Loading CacheModule 'memory' from /usr/local/lib/python3.12/site-packages/ansible/plugins/cache/memory.py
538841 1742797546.63983: trying /usr/local/lib/python3.12/site-packages/ansible/plugins/inventory
538841 1742797546.64055: Loading InventoryModule 'host_list' from /usr/local/lib/python3.12/site-packages/ansible/plugins/inventory/host_list.py
538841 1742797546.64134: Loaded config def from plugin (inventory/script)
538841 1742797546.64139: Loading InventoryModule 'script' from /usr/local/lib/python3.12/site-packages/ansible/plugins/inventory/script.py
538841 1742797546.64191: Loading InventoryModule 'auto' from /usr/local/lib/python3.12/site-packages/ansible/plugins/inventory/auto.py
538841 1742797546.64278: Loaded config def from plugin (inventory/yaml)
538841 1742797546.64284: Loading InventoryModule 'yaml' from /usr/local/lib/python3.12/site-packages/ansible/plugins/inventory/yaml.py
538841 1742797546.64381: Loading InventoryModule 'ini' from /usr/local/lib/python3.12/site-packages/ansible/plugins/inventory/ini.py
538841 1742797546.64781: Loading InventoryModule 'toml' from /usr/local/lib/python3.12/site-packages/ansible/plugins/inventory/toml.py
538841 1742797546.64788: Attempting to use plugin host_list (/usr/local/lib/python3.12/site-packages/ansible/plugins/inventory/host_list.py)
538841 1742797546.64794: Attempting to use plugin script (/usr/local/lib/python3.12/site-packages/ansible/plugins/inventory/script.py)
538841 1742797546.64804: Attempting to use plugin auto (/usr/local/lib/python3.12/site-packages/ansible/plugins/inventory/auto.py)
538841 1742797546.64807: Attempting to use plugin yaml (/usr/local/lib/python3.12/site-packages/ansible/plugins/inventory/yaml.py)
538841 1742797546.64861: Loading data from /root/ansible_modules_redirection_test4/ansible-junos-stdlib/ansible_collections/inventory
538841 1742797546.64932: /root/ansible_modules_redirection_test4/ansible-junos-stdlib/ansible_collections/inventory was not parsable by yaml
538841 1742797546.65015: Attempting to use plugin ini (/usr/local/lib/python3.12/site-packages/ansible/plugins/inventory/ini.py)
538841 1742797546.65089: Added group junos to inventory
538841 1742797546.65103: set inventory_file for 10.220.10.59
538841 1742797546.65115: set inventory_dir for 10.220.10.59
538841 1742797546.65125: Added host 10.220.10.59 to inventory
538841 1742797546.65129: Added host 10.220.10.59 to group junos
538841 1742797546.65147: set ansible_network_os for junos
538841 1742797546.65170: set ansible_ssh_user for junos
538841 1742797546.65187: set ansible_ssh_pass for junos
538841 1742797546.65202: set ansible_ssh_port for junos
538841 1742797546.65214: set ansible_connection for junos
538841 1742797546.65258: set ansible_network_cli_ssh_type for junos
538841 1742797546.65267: set ansible_python_interpreter for all
538841 1742797546.65272: Reconcile groups and hosts in inventory.
538841 1742797546.65284: Group all now contains junos
538841 1742797546.65384: '/usr/local/lib/python3.12/site-packages/ansible/plugins/vars/__init__' skipped due to reserved name
538841 1742797546.65513: trying /usr/local/lib/python3.12/site-packages/ansible/plugins/doc_fragments
538841 1742797546.65576: Loading ModuleDocFragment 'vars_plugin_staging' from /usr/local/lib/python3.12/site-packages/ansible/plugins/doc_fragments/vars_plugin_staging.py
538841 1742797546.65613: Loaded config def from plugin (vars/host_group_vars)
538841 1742797546.65618: Loading VarsModule 'host_group_vars' from /usr/local/lib/python3.12/site-packages/ansible/plugins/vars/host_group_vars.py (found_in_cache=False, class_only=True)
538841 1742797546.65640: trying /usr/local/lib/python3.12/site-packages/ansible/plugins/vars
538841 1742797546.65657: Loading VarsModule 'host_group_vars' from /usr/local/lib/python3.12/site-packages/ansible/plugins/vars/host_group_vars.py (found_in_cache=True, class_only=False)
538841 1742797546.65715: Loading CacheModule 'memory' from /usr/local/lib/python3.12/site-packages/ansible/plugins/cache/memory.py (found_in_cache=True, class_only=False)
538841 1742797546.66057: '/usr/local/lib/python3.12/site-packages/ansible/plugins/connection/__init__' skipped due to reserved name
538841 1742797546.66168: Loading ModuleDocFragment 'connection_pipelining' from /usr/local/lib/python3.12/site-packages/ansible/plugins/doc_fragments/connection_pipelining.py
538841 1742797546.66214: Loaded config def from plugin (connection/local)
538841 1742797546.66219: Loading Connection 'local' from /usr/local/lib/python3.12/site-packages/ansible/plugins/connection/local.py (found_in_cache=False, class_only=True)
538841 1742797546.66577: Loaded config def from plugin (connection/paramiko_ssh)
538841 1742797546.66584: Loading Connection 'paramiko_ssh' from /usr/local/lib/python3.12/site-packages/ansible/plugins/connection/paramiko_ssh.py (found_in_cache=False, class_only=True)
538841 1742797546.67397: Loading ModuleDocFragment 'connection_pipelining' from /usr/local/lib/python3.12/site-packages/ansible/plugins/doc_fragments/connection_pipelining.py (found_in_cache=True, class_only=False)
538841 1742797546.67446: Loaded config def from plugin (connection/psrp)
538841 1742797546.67452: Loading Connection 'psrp' from /usr/local/lib/python3.12/site-packages/ansible/plugins/connection/psrp.py (found_in_cache=False, class_only=True)
538841 1742797546.67978: Loading ModuleDocFragment 'connection_pipelining' from /usr/local/lib/python3.12/site-packages/ansible/plugins/doc_fragments/connection_pipelining.py (found_in_cache=True, class_only=False)
538841 1742797546.68026: Loaded config def from plugin (connection/ssh)
538841 1742797546.68063: Loading Connection 'ssh' from /usr/local/lib/python3.12/site-packages/ansible/plugins/connection/ssh.py (found_in_cache=False, class_only=True)
538841 1742797546.69672: Loading ModuleDocFragment 'connection_pipelining' from /usr/local/lib/python3.12/site-packages/ansible/plugins/doc_fragments/connection_pipelining.py (found_in_cache=True, class_only=False)
538841 1742797546.69725: Loaded config def from plugin (connection/winrm)
538841 1742797546.69732: Loading Connection 'winrm' from /usr/local/lib/python3.12/site-packages/ansible/plugins/connection/winrm.py (found_in_cache=False, class_only=True)
538841 1742797546.69782: '/usr/local/lib/python3.12/site-packages/ansible/plugins/shell/__init__' skipped due to reserved name
538841 1742797546.69858: Loading ModuleDocFragment 'shell_windows' from /usr/local/lib/python3.12/site-packages/ansible/plugins/doc_fragments/shell_windows.py
538841 1742797546.69918: Loaded config def from plugin (shell/cmd)
538841 1742797546.69924: Loading ShellModule 'cmd' from /usr/local/lib/python3.12/site-packages/ansible/plugins/shell/cmd.py (found_in_cache=False, class_only=True)
538841 1742797546.69957: Loading ModuleDocFragment 'shell_windows' from /usr/local/lib/python3.12/site-packages/ansible/plugins/doc_fragments/shell_windows.py (found_in_cache=True, class_only=False)
538841 1742797546.70016: Loaded config def from plugin (shell/powershell)
538841 1742797546.70022: Loading ShellModule 'powershell' from /usr/local/lib/python3.12/site-packages/ansible/plugins/shell/powershell.py (found_in_cache=False, class_only=True)
538841 1742797546.70105: Loading ModuleDocFragment 'shell_common' from /usr/local/lib/python3.12/site-packages/ansible/plugins/doc_fragments/shell_common.py
538841 1742797546.70243: Loaded config def from plugin (shell/sh)
538841 1742797546.70249: Loading ShellModule 'sh' from /usr/local/lib/python3.12/site-packages/ansible/plugins/shell/sh.py (found_in_cache=False, class_only=True)
538841 1742797546.70304: '/usr/local/lib/python3.12/site-packages/ansible/plugins/become/__init__' skipped due to reserved name
538841 1742797546.70422: Loaded config def from plugin (become/runas)
538841 1742797546.70430: Loading BecomeModule 'runas' from /usr/local/lib/python3.12/site-packages/ansible/plugins/become/runas.py (found_in_cache=False, class_only=True)
538841 1742797546.70614: Loaded config def from plugin (become/su)
538841 1742797546.70620: Loading BecomeModule 'su' from /usr/local/lib/python3.12/site-packages/ansible/plugins/become/su.py (found_in_cache=False, class_only=True)
538841 1742797546.70770: Loaded config def from plugin (become/sudo)
538841 1742797546.70776: Loading BecomeModule 'sudo' from /usr/local/lib/python3.12/site-packages/ansible/plugins/become/sudo.py (found_in_cache=False, class_only=True)
538841 1742797546.70822: Loading data from /root/ansible_modules_redirection_test4/ansible-junos-stdlib/ansible_collections/device_facts.yml
538841 1742797546.71301: trying /usr/local/lib/python3.12/site-packages/ansible/plugins/callback
538841 1742797546.71408: Loading ModuleDocFragment 'default_callback' from /usr/local/lib/python3.12/site-packages/ansible/plugins/doc_fragments/default_callback.py
538841 1742797546.71563: Loading ModuleDocFragment 'result_format_callback' from /usr/local/lib/python3.12/site-packages/ansible/plugins/doc_fragments/result_format_callback.py
538841 1742797546.71633: Loaded config def from plugin (callback/default)
538841 1742797546.71639: Loading CallbackModule 'default' from /usr/local/lib/python3.12/site-packages/ansible/plugins/callback/default.py
538841 1742797546.71700: '/usr/local/lib/python3.12/site-packages/ansible/plugins/callback/__init__' skipped due to reserved name
538841 1742797546.71706: Loading CallbackModule 'default' from /usr/local/lib/python3.12/site-packages/ansible/plugins/callback/default.py (found_in_cache=True, class_only=True)
538841 1742797546.72590: Loaded config def from plugin (callback/junit)
538841 1742797546.72597: Loading CallbackModule 'junit' from /usr/local/lib/python3.12/site-packages/ansible/plugins/callback/junit.py (found_in_cache=False, class_only=True)
538841 1742797546.72656: Loading ModuleDocFragment 'result_format_callback' from /usr/local/lib/python3.12/site-packages/ansible/plugins/doc_fragments/result_format_callback.py (found_in_cache=True, class_only=False)
538841 1742797546.72719: Loaded config def from plugin (callback/minimal)
538841 1742797546.72725: Loading CallbackModule 'minimal' from /usr/local/lib/python3.12/site-packages/ansible/plugins/callback/minimal.py (found_in_cache=False, class_only=True)
538841 1742797546.72778: Loading CallbackModule 'oneline' from /usr/local/lib/python3.12/site-packages/ansible/plugins/callback/oneline.py (found_in_cache=False, class_only=True)
538841 1742797546.72847: Loaded config def from plugin (callback/tree)
538841 1742797546.72853: Loading CallbackModule 'tree' from /usr/local/lib/python3.12/site-packages/ansible/plugins/callback/tree.py (found_in_cache=False, class_only=True)
538841 1742797546.72870: in VariableManager get_vars()
538841 1742797546.72937: done with get_vars()
538841 1742797546.72951: in VariableManager get_vars()
538841 1742797546.72977: done with get_vars()
538841 1742797546.72993: variable 'omit' from source: magic vars
538841 1742797546.73037: in VariableManager get_vars()
538841 1742797546.73058: done with get_vars()
538841 1742797546.73093: variable 'omit' from source: magic vars

PLAY [Test Junos Facts Module] ******************************************************************************************************************************
538841 1742797546.77906: trying /usr/local/lib/python3.12/site-packages/ansible/plugins/strategy
538841 1742797546.78005: Loading StrategyModule 'linear' from /usr/local/lib/python3.12/site-packages/ansible/plugins/strategy/linear.py
538841 1742797546.78074: getting the remaining hosts for this loop
538841 1742797546.78079: done getting the remaining hosts for this loop
538841 1742797546.78082: getting the next task for host 10.220.10.59
538841 1742797546.78092: done getting next task for host 10.220.10.59
538841 1742797546.78100:  ^ task is: TASK: Gather Junos Facts
538841 1742797546.78104:  ^ state is: HOST STATE: block=2, task=1, rescue=0, always=0, handlers=0, run_state=1, fail_state=0, pre_flushing_run_state=None, update_handlers=True, pending_setup=False, tasks child state? (None), rescue child state? (None), always child state? (None), did rescue? False, did start at task? False
538841 1742797546.78118: getting variables
538841 1742797546.78126: in VariableManager get_vars()
538841 1742797546.78155: Calling all_inventory to load vars for 10.220.10.59
538841 1742797546.78162: Calling groups_inventory to load vars for 10.220.10.59
538841 1742797546.78170: Calling all_plugins_inventory to load vars for 10.220.10.59
538841 1742797546.78201: Calling all_plugins_play to load vars for 10.220.10.59
538841 1742797546.78213: Calling groups_plugins_inventory to load vars for 10.220.10.59
538841 1742797546.78219: Calling groups_plugins_play to load vars for 10.220.10.59
538841 1742797546.78277: '/usr/local/lib/python3.12/site-packages/ansible/plugins/connection/__init__' skipped due to reserved name
538841 1742797546.78334: done with get_vars()
538841 1742797546.78345: done getting variables
538841 1742797546.78702: Loading ActionModule 'ansible_collections.juniper.device.plugins.action.facts' from /root/ansible_modules_redirection_test4/ansible-junos-stdlib/ansible_collections/juniper/device/plugins/action/facts.py (found_in_cache=False, class_only=True)

TASK [Gather Junos Facts] ***********************************************************************************************************************************
538841 1742797546.78730: entering _queue_task() for 10.220.10.59/junos_facts
538841 1742797546.78742: Creating lock for junos_facts
538841 1742797546.79046: worker is 1 (out of 1 available)
538841 1742797546.79094: exiting _queue_task() for 10.220.10.59/junos_facts
538841 1742797546.79119: done queuing things up, now waiting for results queue to drain
538841 1742797546.79124: waiting for pending results...
538841 1742797546.79517: running TaskExecutor() for 10.220.10.59/TASK: Gather Junos Facts
538841 1742797546.79667: in run() - task 54040adc-1369-a18d-b47a-000000000004
538841 1742797546.79700: variable 'ansible_search_path' from source: unknown
538841 1742797546.79784: calling self._execute()
538841 1742797546.79878: variable 'ansible_connection' from source: group vars, precedence entry 'groups_inventory'
538841 1742797546.79892: variable 'ansible_ssh_user' from source: group vars, precedence entry 'groups_inventory'
538841 1742797546.79905: variable 'ansible_ssh_pass' from source: group vars, precedence entry 'groups_inventory'
538841 1742797546.79916: variable 'ansible_ssh_port' from source: group vars, precedence entry 'groups_inventory'
538841 1742797546.79927: variable 'ansible_network_os' from source: group vars, precedence entry 'groups_inventory'
538841 1742797546.79933: variable 'omit' from source: magic vars
538841 1742797546.80122: variable 'omit' from source: magic vars
538841 1742797546.80177: variable 'omit' from source: magic vars
538841 1742797546.80228: variable 'omit' from source: magic vars
538841 1742797546.80256: variable 'ansible_connection' from source: group vars, precedence entry 'groups_inventory'
538841 1742797546.80270: variable 'ansible_connection' from source: group vars, precedence entry 'groups_inventory'
538841 1742797546.84558: Loading ModuleDocFragment 'ansible_collections.ansible.netcommon.plugins.doc_fragments.connection_persistent' from /usr/local/lib/python3.12/site-packages/ansible_collections/ansible/netcommon/plugins/doc_fragments/connection_persistent.py
538841 1742797546.84670: Loaded config def from plugin (connection/ansible_collections.ansible.netcommon.plugins.connection.netconf)
538841 1742797546.84699: Loading Connection 'ansible_collections.ansible.netcommon.plugins.connection.netconf' from /usr/local/lib/python3.12/site-packages/ansible_collections/ansible/netcommon/plugins/connection/netconf.py
538841 1742797546.84729: trying /usr/local/lib/python3.12/site-packages/ansible/plugins/shell
538841 1742797546.84776: Loading ShellModule 'sh' from /usr/local/lib/python3.12/site-packages/ansible/plugins/shell/sh.py (found_in_cache=True, class_only=False)
538841 1742797546.84802: Loading ShellModule 'sh' from /usr/local/lib/python3.12/site-packages/ansible/plugins/shell/sh.py (found_in_cache=True, class_only=False)
538841 1742797546.84835: trying /usr/local/lib/python3.12/site-packages/ansible/plugins/connection
538841 1742797546.84861: Loading Connection 'local' from /usr/local/lib/python3.12/site-packages/ansible/plugins/connection/local.py (found_in_cache=True, class_only=False)
538841 1742797546.84887: Loading ShellModule 'sh' from /usr/local/lib/python3.12/site-packages/ansible/plugins/shell/sh.py (found_in_cache=True, class_only=False)
538841 1742797546.84910: Loading ShellModule 'sh' from /usr/local/lib/python3.12/site-packages/ansible/plugins/shell/sh.py (found_in_cache=True, class_only=False)
538841 1742797546.84980: Loading Connection 'ssh' from /usr/local/lib/python3.12/site-packages/ansible/plugins/connection/ssh.py (found_in_cache=True, class_only=True)
538841 1742797546.85158: trying /usr/local/lib/python3.12/site-packages/ansible/plugins/netconf
538841 1742797546.85183: trying /usr/local/lib/python3.12/site-packages/ansible/plugins/netconf/__pycache__
538841 1742797546.85821: Loaded config def from plugin (netconf/ansible_collections.ansible.netcommon.plugins.netconf.default)
538841 1742797546.85846: Loading Netconf 'ansible_collections.ansible.netcommon.plugins.netconf.default' from /usr/local/lib/python3.12/site-packages/ansible_collections/ansible/netcommon/plugins/netconf/default.py (searched paths: /usr/local/lib/python3.12/site-packages/ansible/plugins/netconf:/usr/local/lib/python3.12/site-packages/ansible/plugins/netconf/__pycache__)
538841 1742797546.85872: variable 'inventory_hostname' from source: host vars for '10.220.10.59'
538841 1742797546.85894: variable 'ansible_network_os' from source: group vars, precedence entry 'groups_inventory'
538841 1742797546.85909: variable 'ansible_ssh_pass' from source: group vars, precedence entry 'groups_inventory'
538841 1742797546.86019: Set connection var ansible_host to 10.220.10.59
538841 1742797546.86039: Set connection var ansible_pipelining to False
538841 1742797546.86057: Set connection var ansible_shell_executable to /bin/sh
538841 1742797546.86072: Set connection var ansible_module_compression to ZIP_DEFLATED
538841 1742797546.86079: Set connection var ansible_timeout to 10
538841 1742797546.86092: Set connection var ansible_shell_type to sh
538841 1742797546.86104: Set connection var ansible_port to 830
538841 1742797546.86116: variable 'ansible_shell_executable' from source: unknown
538841 1742797546.86123: variable 'ansible_connection' from source: group vars, precedence entry 'groups_inventory'
538841 1742797546.86141: variable 'ansible_module_compression' from source: unknown
538841 1742797546.86159: variable 'ansible_shell_type' from source: unknown
538841 1742797546.86178: variable 'ansible_shell_executable' from source: unknown
538841 1742797546.86192: variable 'ansible_host' from source: unknown
538841 1742797546.86209: variable 'ansible_ssh_user' from source: group vars, precedence entry 'groups_inventory'
538841 1742797546.86224: variable 'ansible_ssh_pass' from source: group vars, precedence entry 'groups_inventory'
538841 1742797546.86242: variable 'ansible_ssh_port' from source: group vars, precedence entry 'groups_inventory'
538841 1742797546.86256: variable 'ansible_pipelining' from source: unknown
538841 1742797546.86270: variable 'ansible_timeout' from source: unknown
538841 1742797546.86285: variable 'ansible_network_os' from source: group vars, precedence entry 'groups_inventory'
538841 1742797546.86332: trying /usr/local/lib/python3.12/site-packages/ansible/plugins/action
538841 1742797546.86397: trying /usr/local/lib/python3.12/site-packages/ansible/plugins/action/__pycache__
538841 1742797547.59591: Loading ActionModule 'ansible_collections.juniper.device.plugins.action.facts' from /root/ansible_modules_redirection_test4/ansible-junos-stdlib/ansible_collections/juniper/device/plugins/action/facts.py (searched paths: /usr/local/lib/python3.12/site-packages/ansible/plugins/action:/usr/local/lib/python3.12/site-packages/ansible/plugins/action/__pycache__) (found_in_cache=True, class_only=False)
538841 1742797547.59670: variable 'omit' from source: magic vars
538841 1742797547.59688: starting attempt loop
538841 1742797547.59700: running the handler
538841 1742797547.59741: variable 'ansible_host' from source: unknown
538841 1742797547.59757: variable 'ansible_ssh_user' from source: group vars, precedence entry 'groups_inventory'
538841 1742797547.59779: variable 'ansible_ssh_pass' from source: group vars, precedence entry 'groups_inventory'
538841 1742797547.59797: variable 'ansible_ssh_port' from source: group vars, precedence entry 'groups_inventory'
538841 1742797547.59820: variable 'ansible_timeout' from source: unknown
538841 1742797547.59849: variable 'inventory_hostname' from source: host vars for '10.220.10.59'
538841 1742797547.59947: _low_level_execute_command(): starting
538841 1742797547.59987: _low_level_execute_command(): executing: /bin/sh -c '( umask 77 && mkdir -p "` echo /root/.ansible/tmp/ansible-local-538841_xrv_i1j `"&& mkdir "` echo /root/.ansible/tmp/ansible-local-538841_xrv_i1j/ansible-tmp-1742797547.5990934-538845-279068288112915 `" && echo ansible-tmp-1742797547.5990934-538845-279068288112915="` echo /root/.ansible/tmp/ansible-local-538841_xrv_i1j/ansible-tmp-1742797547.5990934-538845-279068288112915 `" ) && sleep 0'
538841 1742797547.60013: in local.exec_command()
538841 1742797547.60029: opening command with Popen()
538841 1742797547.60589: done running command with Popen()
538841 1742797547.60610: getting output with communicate()
538841 1742797547.61311: done communicating
538841 1742797547.61336: done with local.exec_command()
538841 1742797547.61346: _low_level_execute_command() done: rc=0, stdout=ansible-tmp-1742797547.5990934-538845-279068288112915=/root/.ansible/tmp/ansible-local-538841_xrv_i1j/ansible-tmp-1742797547.5990934-538845-279068288112915
, stderr=
538841 1742797547.61493: variable 'ansible_module_compression' from source: unknown
538841 1742797547.61563: ANSIBALLZ: Using lock for junos_facts
538841 1742797547.61576: ANSIBALLZ: Acquiring lock
538841 1742797547.61590: ANSIBALLZ: Lock acquired: 140636006927920
538841 1742797547.61602: ANSIBALLZ: Creating module
538841 1742797547.76408: ANSIBALLZ: Writing module into payload
538841 1742797547.76529: ANSIBALLZ: Writing module
538841 1742797547.76582: ANSIBALLZ: Renaming module
538841 1742797547.76624: ANSIBALLZ: Done creating module
538841 1742797547.76664: variable 'ansible_playbook_python' from source: magic vars
538841 1742797547.76725: transferring module to remote /root/.ansible/tmp/ansible-local-538841_xrv_i1j/ansible-tmp-1742797547.5990934-538845-279068288112915/AnsiballZ_facts.py
538841 1742797547.76986: done transferring module to remote
538841 1742797547.77059: _low_level_execute_command(): starting
538841 1742797547.77103: _low_level_execute_command(): executing: /bin/sh -c 'chmod u+x /root/.ansible/tmp/ansible-local-538841_xrv_i1j/ansible-tmp-1742797547.5990934-538845-279068288112915/ /root/.ansible/tmp/ansible-local-538841_xrv_i1j/ansible-tmp-1742797547.5990934-538845-279068288112915/AnsiballZ_facts.py && sleep 0'
538841 1742797547.77133: in local.exec_command()
538841 1742797547.77184: opening command with Popen()
538841 1742797547.77667: done running command with Popen()
538841 1742797547.77687: getting output with communicate()
538841 1742797547.78016: done communicating
538841 1742797547.78034: done with local.exec_command()
538841 1742797547.78047: _low_level_execute_command() done: rc=0, stdout=, stderr=
538841 1742797547.78059: _low_level_execute_command(): starting
538841 1742797547.78080: _low_level_execute_command(): executing: /bin/sh -c '/usr/bin/python3 /root/.ansible/tmp/ansible-local-538841_xrv_i1j/ansible-tmp-1742797547.5990934-538845-279068288112915/AnsiballZ_facts.py && sleep 0'
538841 1742797547.78085: in local.exec_command()
538841 1742797547.78104: opening command with Popen()
538841 1742797547.78406: done running command with Popen()
538841 1742797547.78424: getting output with communicate()
538841 1742797548.89653: done communicating
538841 1742797548.89688: done with local.exec_command()
538841 1742797548.89704: _low_level_execute_command() done: rc=1, stdout=, stderr=Traceback (most recent call last):
  File "/root/.ansible/tmp/ansible-local-538841_xrv_i1j/ansible-tmp-1742797547.5990934-538845-279068288112915/AnsiballZ_facts.py", line 107, in <module>
    _ansiballz_main()
  File "/root/.ansible/tmp/ansible-local-538841_xrv_i1j/ansible-tmp-1742797547.5990934-538845-279068288112915/AnsiballZ_facts.py", line 99, in _ansiballz_main
    invoke_module(zipped_mod, temp_path, ANSIBALLZ_PARAMS)
  File "/root/.ansible/tmp/ansible-local-538841_xrv_i1j/ansible-tmp-1742797547.5990934-538845-279068288112915/AnsiballZ_facts.py", line 47, in invoke_module
    runpy.run_module(mod_name='ansible_collections.juniper.device.plugins.modules.facts', init_globals=dict(_module_fqn='ansible_collections.juniper.device.plugins.modules.facts', _modlib_path=modlib_path),
  File "<frozen runpy>", line 226, in run_module
  File "<frozen runpy>", line 98, in _run_module_code
  File "<frozen runpy>", line 88, in _run_code
  File "/tmp/ansible_junos_facts_payload_qjy3mpx6/ansible_junos_facts_payload.zip/ansible_collections/juniper/device/plugins/modules/facts.py", line 373, in <module>
  File "/tmp/ansible_junos_facts_payload_qjy3mpx6/ansible_junos_facts_payload.zip/ansible_collections/juniper/device/plugins/modules/facts.py", line 317, in main
  File "/tmp/ansible_junos_facts_payload_qjy3mpx6/ansible_junos_facts_payload.zip/ansible_collections/juniper/device/plugins/module_utils/juniper_junos_common.py", line 689, in __init__
  File "/tmp/ansible_junos_facts_payload_qjy3mpx6/ansible_junos_facts_payload.zip/ansible_collections/juniper/device/plugins/module_utils/juniper_junos_common.py", line 762, in get_connection
  File "/tmp/ansible_junos_facts_payload_qjy3mpx6/ansible_junos_facts_payload.zip/ansible_collections/juniper/device/plugins/module_utils/juniper_junos_common.py", line 803, in fail_json
  File "/tmp/ansible_junos_facts_payload_qjy3mpx6/ansible_junos_facts_payload.zip/ansible_collections/juniper/device/plugins/module_utils/juniper_junos_common.py", line 1287, in close_configuration
AttributeError: 'JuniperJunosModule' object has no attribute '_pyez_conn'
538841 1742797548.90057: done with _execute_module (junos_facts, {'_connection': 'ansible.netcommon.netconf', 'user': 'xyz', 'passwd': 'xyz', 'host': '10.220.10.59', 'port': 22, 'timeout': 10, '_module_utils_path': '/root/ansible_modules_redirection_test4/ansible-junos-stdlib/ansible_collections/juniper/device/plugins/action', '_module_name': 'junos_facts', '_inventory_hostname': '10.220.10.59', '_ansible_check_mode': False, '_ansible_no_log': False, '_ansible_debug': True, '_ansible_diff': False, '_ansible_verbosity': 0, '_ansible_version': '2.18.3', '_ansible_module_name': 'junos_facts', '_ansible_syslog_facility': 'LOG_USER', '_ansible_selinux_special_fs': ['fuse', 'nfs', 'vboxsf', 'ramfs', '9p', 'vfat'], '_ansible_string_conversion_action': 'warn', '_ansible_socket': '/root/.ansible/pc/5146e715e8', '_ansible_shell_executable': '/bin/sh', '_ansible_keep_remote_files': False, '_ansible_tmpdir': '/root/.ansible/tmp/ansible-local-538841_xrv_i1j/ansible-tmp-1742797547.5990934-538845-279068288112915/', '_ansible_remote_tmp': '~/.ansible/tmp', '_ansible_ignore_unknown_opts': False, '_ansible_target_log_info': None})
538841 1742797548.90101: _low_level_execute_command(): starting
538841 1742797548.90155: _low_level_execute_command(): executing: /bin/sh -c 'rm -f -r /root/.ansible/tmp/ansible-local-538841_xrv_i1j/ansible-tmp-1742797547.5990934-538845-279068288112915/ > /dev/null 2>&1 && sleep 0'
538841 1742797548.90170: in local.exec_command()
538841 1742797548.90210: opening command with Popen()
538841 1742797548.90623: done running command with Popen()
538841 1742797548.90650: getting output with communicate()
538841 1742797548.90971: done communicating
538841 1742797548.90994: done with local.exec_command()
538841 1742797548.91011: _low_level_execute_command() done: rc=0, stdout=, stderr=
538841 1742797548.91026: handler run complete
538841 1742797548.91087: attempt loop complete, returning result
538841 1742797548.91116: _execute() done
538841 1742797548.91131: dumping result to json
538841 1742797548.91147: done dumping result, returning
538841 1742797548.91164: done running TaskExecutor() for 10.220.10.59/TASK: Gather Junos Facts [54040adc-1369-a18d-b47a-000000000004]
538841 1742797548.91188: sending task result for task 54040adc-1369-a18d-b47a-000000000004
538841 1742797548.91385: done sending task result for task 54040adc-1369-a18d-b47a-000000000004
538841 1742797548.91406: marking 10.220.10.59 as failed
538841 1742797548.91439: marking host 10.220.10.59 failed, current state: HOST STATE: block=2, task=1, rescue=0, always=0, handlers=0, run_state=1, fail_state=0, pre_flushing_run_state=None, update_handlers=True, pending_setup=False, tasks child state? (None), rescue child state? (None), always child state? (None), did rescue? False, did start at task? False
538841 1742797548.91460: ^ failed state is now: HOST STATE: block=2, task=1, rescue=0, always=0, handlers=0, run_state=5, fail_state=2, pre_flushing_run_state=None, update_handlers=True, pending_setup=False, tasks child state? (None), rescue child state? (None), always child state? (None), did rescue? False, did start at task? False
538841 1742797548.91481: getting the next task for host 10.220.10.59
538841 1742797548.91497: host 10.220.10.59 is done iterating, returning
An exception occurred during task execution. To see the full traceback, use -vvv. The error was: AttributeError: 'JuniperJunosModule' object has no attribute '_pyez_conn'
538841 1742797548.91610: WORKER PROCESS EXITING
fatal: [10.220.10.59]: FAILED! => {"changed": false, "module_stderr": "Traceback (most recent call last):\n  File \"/root/.ansible/tmp/ansible-local-538841_xrv_i1j/ansible-tmp-1742797547.5990934-538845-279068288112915/AnsiballZ_facts.py\", line 107, in <module>\n    _ansiballz_main()\n  File \"/root/.ansible/tmp/ansible-local-538841_xrv_i1j/ansible-tmp-1742797547.5990934-538845-279068288112915/AnsiballZ_facts.py\", line 99, in _ansiballz_main\n    invoke_module(zipped_mod, temp_path, ANSIBALLZ_PARAMS)\n  File \"/root/.ansible/tmp/ansible-local-538841_xrv_i1j/ansible-tmp-1742797547.5990934-538845-279068288112915/AnsiballZ_facts.py\", line 47, in invoke_module\n    runpy.run_module(mod_name='ansible_collections.juniper.device.plugins.modules.facts', init_globals=dict(_module_fqn='ansible_collections.juniper.device.plugins.modules.facts', _modlib_path=modlib_path),\n  File \"<frozen runpy>\", line 226, in run_module\n  File \"<frozen runpy>\", line 98, in _run_module_code\n  File \"<frozen runpy>\", line 88, in _run_code\n  File \"/tmp/ansible_junos_facts_payload_qjy3mpx6/ansible_junos_facts_payload.zip/ansible_collections/juniper/device/plugins/modules/facts.py\", line 373, in <module>\n  File \"/tmp/ansible_junos_facts_payload_qjy3mpx6/ansible_junos_facts_payload.zip/ansible_collections/juniper/device/plugins/modules/facts.py\", line 317, in main\n  File \"/tmp/ansible_junos_facts_payload_qjy3mpx6/ansible_junos_facts_payload.zip/ansible_collections/juniper/device/plugins/module_utils/juniper_junos_common.py\", line 689, in __init__\n  File \"/tmp/ansible_junos_facts_payload_qjy3mpx6/ansible_junos_facts_payload.zip/ansible_collections/juniper/device/plugins/module_utils/juniper_junos_common.py\", line 762, in get_connection\n  File \"/tmp/ansible_junos_facts_payload_qjy3mpx6/ansible_junos_facts_payload.zip/ansible_collections/juniper/device/plugins/module_utils/juniper_junos_common.py\", line 803, in fail_json\n  File \"/tmp/ansible_junos_facts_payload_qjy3mpx6/ansible_junos_facts_payload.zip/ansible_collections/juniper/device/plugins/module_utils/juniper_junos_common.py\", line 1287, in close_configuration\nAttributeError: 'JuniperJunosModule' object has no attribute '_pyez_conn'\n", "module_stdout": "", "msg": "MODULE FAILURE: No start of json char found\nSee stdout/stderr for the exact error", "rc": 1}
538841 1742797548.91713: no more pending results, returning what we have
538841 1742797548.91732: results queue empty
538841 1742797548.91746: checking for any_errors_fatal
538841 1742797548.91762: done checking for any_errors_fatal
538841 1742797548.91772: checking for max_fail_percentage
538841 1742797548.91787: done checking for max_fail_percentage
538841 1742797548.91796: checking to see if all hosts have failed and the running result is not ok
538841 1742797548.91803: done checking to see if all hosts have failed
538841 1742797548.91812: getting the remaining hosts for this loop
538841 1742797548.91820: done getting the remaining hosts for this loop
538841 1742797548.91828: getting the next task for host 10.220.10.59
538841 1742797548.91840: host 10.220.10.59 is done iterating, returning
538841 1742797548.91852: done queuing things up, now waiting for results queue to drain
538841 1742797548.91868: results queue empty
538841 1742797548.91881: checking for any_errors_fatal
538841 1742797548.91893: done checking for any_errors_fatal
538841 1742797548.91903: checking for max_fail_percentage
538841 1742797548.91914: done checking for max_fail_percentage
538841 1742797548.91924: checking to see if all hosts have failed and the running result is not ok
538841 1742797548.91931: done checking to see if all hosts have failed
538841 1742797548.91939: getting the next task for host 10.220.10.59
538841 1742797548.91946: host 10.220.10.59 is done iterating, returning

PLAY RECAP **************************************************************************************************************************************************
10.220.10.59               : ok=0    changed=0    unreachable=0    failed=1    skipped=0    rescued=0    ignored=0   

